google/ads/google_ads/client.py

```diff
@@ -94,6 +94,12 @@ def load_from_string(cls, yaml_str):
         config_data = config.parse_yaml_document_to_dict(yaml_str)
         kwargs = cls._get_client_kwargs(config_data)
         return cls(**kwargs)
+    @classmethod  
+    def load_from_dict(cls,config_dict):
+        
+	config_data=config.load_from_dict(config_dict)
+	kwargs = cls._get_client_kwargs(config_data)
+        return cls(**kwargs)
```

Comment:
The indentation here looks off, please make sure these lines are all indented by 4 spaces.